src/Evolution/Executables/GrMhd/ValenciaDivClean/EvolveValenciaDivCleanBlastWaveFwd.hpp

```diff
@@ -0,0 +1,6 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+struct EvolutionMetavars;
```

Comment:
For our understanding, why is this file necessary? In other cases, it would seem preferable to just forward-declare the single struct.

Reply:
It is needed for `cmake/AddSpectreExecutable.cmake` to generate the actual executable.  It is only necessary when the metavariables are a template, but since we expect most systems to be templated on dimension, it was decided to do it this way for all systems rather than support two setups.